data/mods/Magiclysm/ui/sidebar.json

```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "id": "mana_num",
+    "type": "widget",
+    "label": "Mana",
+    "style": "number",
+    "var": "mana",
+    "colors": [ "c_red", "c_light_red", "c_yellow", "c_light_blue" ]
+  },
+  {
+    "id": "max_mana_num",
+    "type": "widget",
+    "label": "Max Mana",
+    "style": "number",
+    "var": "max_mana",
+    "colors": [ "c_light_blue" ]
+  },
+  {
+    "id": "mana_graph",
+    "type": "widget",
+    "label": "Mana",
+    "style": "graph",
+    "width": 25,
+    "symbols": ".-=#",
+    "fill": "pool",
```

Comment:
I'm not married to the width or symbols, but I sort of like the "pool" style for this, since it gives a good sense of the physical change in mana level with each spell cast.

Reply:
I went ahead and included both "Mana Graph" and "Mana Pool" layouts (see screenshots).